packages/admin/resources/views/components/layouts/app/sidebar/item.blade.php

```diff
@@ -11,6 +11,21 @@
         href="{{ $url }}"
         {!! $shouldOpenUrlInNewTab ? 'target="_blank"' : '' !!}
         x-on:click="window.matchMedia(`(max-width: 1024px)`).matches && $store.sidebar.close()"
+        x-data="{tooltip: {}}"
+        x-init="
+            Alpine.effect(() => {
+                if (Alpine.store('sidebar').isOpen) {
+                    tooltip = false
+                } else {
+                    tooltip = {
+                        content: '{{ $slot }}',
```

Comment:
```suggestion
                        content: @js($slot),
```

Reply:
You should be able to do something like `@js((string) $slot)`

Reply:
Slots are `HtmlString`s. I could do `@js($slot->toHtml())` if that's better.

Reply:
Yeah, either or. I'm pretty sure `HtmlString` has a `__toString()` method as well, but if not yeah, `toHtml()`.

Reply:
Casting or `->toHtml()` what do you prefer? I think I'd go with the more explicit function.

Reply:
Yeah, `toHtml()` then.